smartapps/smartthings/hue-connect.src/hue-connect.groovy

```diff
@@ -1182,9 +1182,8 @@ private poll() {
 	def host = getBridgeIP()
 	def uri = "/api/${state.username}/lights/"
 	log.debug "GET: $host$uri"
-	sendHubCommand(new physicalgraph.device.HubAction("""GET ${uri} HTTP/1.1
-HOST: ${host}
-""", physicalgraph.device.Protocol.LAN, selectedHue))
+	sendHubCommand(new physicalgraph.device.HubAction("GET ${uri} HTTP/1.1\r\n" +
```

Comment:
You may want to use the alternate signature for `HubAction` that takes a Map instead of a string so you don't have to explicitly write the newlines. See `discoverHueBulbs` for a function that does basically the same thing as `poll`.